src/Compilers/CSharp/Test/Emit/PDB/PDBTests.cs

```diff
@@ -7318,6 +7318,42 @@ public void SyntaxOffset_Pattern()
 </symbols>");
         }
 
+        [ConditionalFact(typeof(WindowsOnly), Reason = ConditionalSkipReason.NativePdbRequiresDesktop)]
```

Comment:
It's possible that we fixed the PDB testing infrastructure to support Core too. Try with `[Fact]`?

Reply:
I'll follow up and flip all tests.